doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,230 @@
+| manage_vlan_id | String | Yes | may multiple vlan_id for management plane. Setting the dedicated vlan id assigned.|
+| storage_vlan_id | String | Yes | may multiple vlan_id for storage plane. Setting the dedicated vlan id assigned.|
+
+<p align="center"><b>Table 5-3-6:</b> Controller information.</p>
+
+### 5.3.7 Compute informations
+List of compute nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory? | Instruction |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| device_name | String | Yes | must be the device_name defined in server_information(###5.3.3), which would be set on Server from BMC|
+| node_name | String | Yes | node name for the compute to deploy, e.g: compute01 |
+| az_name | String | Yes | availability zone name which this node belongs to |
+| ha_name | List | Yes | HA name which this node assigned to, could be more than one HA it belongs.|
+| manage_vlan_id | String | Yes | may multiple vlan_id for management plane. Setting the dedicated vlan id assigned.|
+| storage_vlan_id | String | Yes | may multiple vlan_id for storage plane. Setting the dedicated vlan id assigned.|
+| vnic_type | String | Yes | VIM verification needs know compute node type: OVS,DPDK_OVS or SR-IOV node.|
+| os_reserved_cores | String | No | Designed and planned by provider itself, may be none|
+| ovs_reserved_cores | String | No | Designed and planned by provider itself, may be none |
+
+<p align="center"><b>Table 5-3-7:</b> Compute information.</p>
+
+### 5.3.8 Distributed storage informations
+List of compute nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory? | Instruction |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster_infos | List | Yes | a list of cluster information|
+<p align="center"><b>Table 5-3-8-1:</b> Storage information.</p>
+
+| Field # | type | mandatory? | Instruction |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster_name | String | Yes | indicate the cluster name|
+| ceph_iscsi_gateway | String | Yes | splitted by semicolon if more than one. e.g 2409:8086:8412:10a::1;2409:8086:8412:10a::2|
+| ceph_manage_plane_address_segment | String | Yes | The address segment of ceph management plane. e.g 2409:8086:8412:100::/64|
+| ceph_storage_plane_address_segment | String | Yes | The address segment of ceph storage plane. e.g 2409:8086:8412:100::/64|
+| ceph_os_password | String | Yes | the tecs user's password of the ceph OS. |
```

Comment:
Should this be TECS specific?

Reply:
that's the user name, tecs which has been created by default. To adapt a common way, I would add a ceph_os_user as well.

Reply:
'ceph_os_user' change is done?